app/components/search_component.html.erb

```diff
@@ -0,0 +1,20 @@
+<%= search_form_for @q, url: @url, html: {"data-controller": "filters"} do |f| %>
+  <input type="hidden" name="format" value="turbo_stream"/>
+  <input type="hidden" name="tab" value="<%=@tab%>"/>
+  <%= f.label @search_attribute, @placeholder, class: "sr-only" %>
+  <div class="relative lg:w-72">
+    <div
+      class="
+        absolute inset-y-0 left-0 flex items-center pl-3 pointer-events-none
+      "
+    >
+      <%= viral_icon(name: "magnifying_glass", classes: "h-5 w-5") %>
+    </div>
+    <%= helpers.turbo_frame_tag "member_hidden_values" %>
```

Comment:
I think we should update this turbo_frame_tag to `ransack_hidden_values` and then update wherever we use `member_hidden_values` since it's just adding ransack hidden sort values

Reply:
3dbb985ad983a1d56580a16133a6aa71cb67ed2c